### PR TITLE
Add Main Menu

### PR DESCRIPTION
**Pull request by ExclusiveKnight**
I have created a simple main menu screen for the game with background music also there is a Sounds folder where I put the main menu music in. I made it into a main scene so whenever you launch the game, the main menu will be the current scene.

### CODE REVIEW

**Comment by xix-xeaon** [comment]
I'm not sure if we need an AudioManager, it's not part of the minimum viable project plan. Update this code to include only the main menu. We might have use of an AudioManger in the future though.

**Review by Snurrigt1 — COMMENTED** [review]
Looks good for a main menu

File: Menu/Main/main-menu.tscn [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+offset_top = -124.0
+offset_right = 173.0
+offset_bottom = -79.0
+grow_horizontal = 2
+grow_vertical = 2
+theme_override_colors/font_shadow_color = Color(0, 0, 0, 1)
+theme_override_constants/shadow_offset_x = 3
+theme_override_constants/shadow_outline_size = 5
+theme_override_font_sizes/font_size = 32
+text = "Created By NTI Malmö"
```
Comment (xix-xeaon) [inline Menu/Main/main-menu.tscn:72]: This should include "IT23" somehow.

**Review by xix-xeaon — CHANGES_REQUESTED** [review]
Just two small changes, then I think we can merge.

You do however need to learn to **make way fewer commits**, even 10 commits for this change would be too many and you're way past that.

File: project.godot
[the remark this answers is not in the document]
Comment (ExclusiveKnight) [reply project.godot:1]: me and the others from my class agreed to make the main menu the main scene but okay

[the remark this answers is not in the document]
Comment (xix-xeaon) [reply project.godot:1]: I was not aware of this. It seems you've updated the PR with this information now so it's okay to make it the main scene!

[the remark this answers is not in the document]
Comment (ExclusiveKnight) [reply project.godot:1]: I'll re-add it when I have time because the main menu is no longer the project's main scene in my latest commit.

**Review by xix-xeaon — APPROVED** [review]

**Pull request merged**